### PR TITLE
 Provision node and run SI tests with pipenv. 

### PR DESCRIPTION
**Pull request by jeschkies**
Summary:
This introduces pipenv to manage our system integration test dependencies
and environment. This way we can introduces and remove dependencies
without changing the Jenkins Docker images and maintain reproducible runs.

An environment is initialized with `make init`.
The tests are run with with `make test`.

You can also jump into the shell of the environment with `pipenv shell`.

### CODE REVIEW

**Comment by jeschkies** [comment]
[This](https://jenkins.mesosphere.com/service/jenkins/view/Marathon/job/system-integration-tests/job/marathon-si-dcos-open/job/karsten%252Fpipenv/14/) is an example job. Bundling does not work yet.

File: ci/launch_cluster.sh [hunk 1]
```diff
@@ -24,8 +24,9 @@ fi
 echo "Workspace: ${WORKSPACE}"
 echo "Using: ${TEMPLATE}"
 
-apk update
-apk --upgrade add gettext wget
+# apk update
```
Comment (alenkacz) [inline ci/launch_cluster.sh]: why did we just commented out this?

Comment (jeschkies) [reply ci/launch_cluster.sh]: Ah, this should go.

**Comment by jeschkies** [comment]
I've also triggered a test run against a [strict cluster](https://jenkins.mesosphere.com/service/jenkins/view/Marathon/job/system-integration-tests/job/marathon-si-dcos-strict/job/karsten%252Fpipenv/).

**Pull request merged**